tests/lib/nested.sh

```diff
@@ -1040,7 +1043,12 @@ nested_start_core_vm_unit() {
 
     local PARAM_MACHINE
     if [[ "$SPREAD_BACKEND" = google-nested* ]]; then
-        PARAM_MACHINE="-machine ubuntu${ATTR_KVM}"
+        if os.query is-arm; then
+            PARAM_MACHINE="-machine virt${ATTR_KVM}"
+            PARAM_CPU="-cpu cortex-a57"
```

Comment:
Maybe we should use "host", see https://qemu-project.gitlab.io/qemu/system/arm/cpu-features.html#a-note-about-cpu-models-and-kvm

Reply:
Yesn thanks for the suggestion. Updated